frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -72,8 +72,8 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'FeatureFlag/Action',
     properties: {
-      flag: OCS_VERSION_4_5_FLAG,
-      detect: detectOCSVersion45AndAbove,
+      flag: OCS_SUPPORT_FLAGS.OCS_SNAPSHOT,
```

Comment:
Not required anymore since https://github.com/openshift/console/pull/5114 has merged.